roles/privacyidea/templates/pi.cfg.j2

```diff
@@ -0,0 +1,26 @@
+
+#https://github.com/privacyidea/privacyidea/blob/master/deploy/pi.cfg
+import logging
+# The realm, where users are allowed to login as administrators,those realm are retrieved from the SUPERUSER_REALM and POLICY in Admin2go
```

Comment:
again admin2go?

Reply:
The comment has been adjusted, and it should now reference the dump file